scipy/fftpack/tests/test_czt.py

```diff
@@ -0,0 +1,70 @@
+# This program is public domain
+# Authors: Paul Kienzle, Nadav Horesh
+'''
+  A unit test module for czt.py
+'''
+
+from numpy.testing import run_module_suite
+from czt import czt, zoomfft, scaledfft
+import numpy as np
+
+fft = np.fft.fft
+fftshift = np.fft.fftshift
+norm = np.linalg.norm
+
+
+def check_zoomfft(x):
```

Comment:
functions starting with test_ get called automatically with no arguments?  they should probably be converted into functions of a test class?